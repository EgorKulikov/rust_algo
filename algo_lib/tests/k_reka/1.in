4 0
3 5 5 4
5
1 1
2 1
1 3
2 2
1 3
