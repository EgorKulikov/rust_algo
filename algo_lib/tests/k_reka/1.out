75
105
73
101
83
113
